galaxy_ng/app/api/ui/serializers/collection.py

```diff
@@ -121,31 +121,21 @@ def get_namespace(self, obj):
     def get_deprecated(self, obj):
         return obj.collection.deprecated
 
-    # TODO(awcrosby): refactor once pulp_ansible is_highest param filters by repo
-    # https://pulp.plan.io/issues/7428
     def _get_versions_in_repo(self, obj):
```

Comment:
Feels little odd to define _get_versions_in_repo in this class, but only use it in CollectionDetailSerializer.
But shrug, _CollectionSerializer is always subclass so doesn't matter a ton.